index.js

```diff
@@ -125,22 +159,50 @@ BemConfig.prototype.library = function(libName) {
 };
 
 /**
+ * Returns config for given library synchronously
+ * @param {String} libName - library name
+ * @returns {Object}
+ */
+BemConfig.prototype.librarySync = function(libName) {
+    return new BemConfig({ projectRoot: this.getSync().libs[libName].path });
+};
+
+/**
  * Returns map of settings for each of level
  * @returns {Promise}
  */
 BemConfig.prototype.levelMap = function() {
     return this.get().then(config => {
         const projectLevels = config.levels;
 
-        return Promise.all(Object.keys(config.libs).map(libName => {
+        return Promise.all((config.libs ? Object.keys(config.libs) : []).map(libName => {
```

Comment:
Not really, we are humans. We are not ideal. But we can write the ideal code. We are just lazy also, so nvm.